src/Build/Logging/FileLogger.cs

```diff
@@ -81,6 +81,11 @@ private void InitializeFileLogger(IEventSource eventSource, int nodeCount)
             // want to make decisions based on our verbosity, so we do this last.
             base.Initialize(eventSource, nodeCount);
 
+            if (!SkipProjectStartedText && Verbosity >= LoggerVerbosity.Normal)
+            {
+                eventSource.BuildStarted += (obj, args) => WriteHandler($"Verbosity is set to: {Verbosity}.");
```

Comment:
I think this is a reasonable way to do this right now. I think in the future we may wish to combine this message with a solution to #2810, but we don't have to solve that right now.

Reply:
Do you point me to the next issue that I could do? 😉

Reply:
I'd recommend looking at the [up-for-grabs label](https://github.com/microsoft/msbuild/labels/up-for-grabs) and seeing if anything catches your eye.

Reply:
Sure, I do exactly that, when seeking for new work 😄